src/core/pointcloud/qgspointcloudlayerexporter.h

```diff
@@ -227,7 +265,7 @@ class CORE_EXPORT QgsPointCloudLayerExporter SIP_NODEFAULTCTORS
 
     QgsPointCloudIndex *mIndex = nullptr;
     QString mName = QObject::tr( "Exported" );
-    QString mFormat;
+    ExportFormat mFormat;
```

Comment:
It seems the format value stays uninitialized - would be good to set some valid initial value